### PR TITLE
feat: recovery dashboard widget

### LINKED ISSUE #2753: [Recovery] Dashboard enable widget
## What is the feature about

Widget to enable recovery via the dashboard.

## The list of requirements

- [x] Create widget

## Designs/sketches

image

## Links

- [Feature spec.](https://www.notion.so/safe-global/Recovery-feature-spec-d873c33a27c24cfabc91c85dbcf3ea35)
- [Figma](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Generic-recovery?type=design&node-id=446-5810&mode=design&t=RGkXuGwJrKf50WY0-0)

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves #2753

## How this PR fixes it

The "New in Safe{Wallet}" section of the dashboard has been updated with a recovery featuring widget as per [design](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Generic-recovery?type=design&node-id=446-5810&mode=design&t=YDv7oOSlANtJWwoM-0). A new `RELAYING` feature has also been added on the staging Client Gateway, enabled on Goerli, which display of the widget depends on.

Note: the button has no functionality until 2760 is implemented.

## How to test it

Open Goerli on staging and observe the new widget.

## Screenshots

image

image

## Checklist
* [x] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: cypress/e2e/safe-apps/tx-builder.spec.cy.js [hunk 1]
```diff
@@ -146,8 +146,8 @@ describe('Tx-builder Safe App tests', { defaultCommandTimeout: 20000 }, () => {
       getBody().findByText(safeapps.createBatchStr).click()
       getBody().findByText(safeapps.sendBatchStr).click()
     })
-    cy.get('p').contains('1').should('be.visible')
-    cy.get('p').contains('2').should('be.visible')
+    cy.get('p').contains('1').should('exist')
+    cy.get('p').contains('2').should('exist')
```
Comment (iamacook) [inline cypress/e2e/safe-apps/tx-builder.spec.cy.js:150]: This was failing for some reason. Fixed after consultation with @mike10ca.

**Comment by liliya-soroka** [comment]
incorrect color for the New label
image
 design:
image

**Comment by iamacook** [comment]
> incorrect color for the New label image design: image

I've updated the colour in bd7fa18cb1c556af168b5e70bde06b1f85802f7b.

File: src/components/dashboard/Recovery/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          <Grid container className={css.grid}>
+            <Grid item>
+              <RecoveryLogo alt="Vault with a circular arrow around it" />
+            </Grid>
+            <Grid item xs>
+              <Box className={css.wrapper}>
+                <Typography variant="h4" className={css.title}>
+                  Introducing account recovery{' '}
+                </Typography>
+                <Chip label="New" color={isDarkMode ? 'primary' : 'secondary'} size="small" className={css.chip} />
```
Comment (usame-algan) [inline src/components/dashboard/Recovery/index.tsx:34]: I suggest we create a `static-primary` color in our theme the same way we have `static-main` so we don't have to use a hook here. Lets do it as a separate PR though.

File: src/components/dashboard/Recovery/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  const isDarkMode = useDarkMode()
+
+  const onClick = () => {
+    // TODO: Open enable recovery flow
+  }
+
+  return (
+    <WidgetContainer>
+      <Typography component="h2" variant="subtitle1" className={css.label}>
+        New in {'Safe{Wallet}'}
```
Comment (usame-algan) [inline src/components/dashboard/Recovery/index.tsx:20]: nit: The Figma design says `New in Safe` here

File: src/components/dashboard/Relaying/index.tsx
Comment (usame-algan) [inline src/components/dashboard/Relaying/index.tsx:1]: I don't mind removing this widget but for visibility I would suggest we document it somewhere.

**Review by usame-algan — APPROVED** [review]
👍

File: src/components/dashboard/Recovery/index.tsx (hunk 3 above)
Comment (iamacook) [reply src/components/dashboard/Recovery/index.tsx:20]: The wording is incorrect in the design. This is a Safe{Wallet} feature.
